### ISSUE TITLE
负采样问题

### ISSUE THREAD

**Issue by LZ-LAB** [liuyuaa/RAM#2]
Yu Liu，你好。我想请教一下，文中提出的n-ary知识图谱的负采样在代码中哪一块体现？

**Comment by liuyuaa** [comment]
In practice, we did the negative sampling on all entities, i.e., all other entities except the predicted one are viewed as negative samples, which comes to the softmax-based loss function here: https://github.com/liuyuaa/RAM/blob/fb4de01ef1a8d26ce372e9c73b932b9928b6876b/model.py#L13.
